### PR TITLE
[Move] GMAX moves weather implemented

### PR DESCRIPTION
**Pull request by DTZK**
## What are the changes?

I added weather conditions added from Gmax moves
## Why am I doing these changes?

I found out that although gmax moves were in the code, weather conditions were not set.

no

The gmax moves effects were not added in so I decided to add it in.

## What did change?

Added hail to MAX_HAILSTORM, added sandstorm in MAX_ROCKFALL, sunny in max flare, and rain in max gyser

### Screenshots/Videos

proof

## How to test the changes?

no

no

no

## Checklist
- [ x] There is no overlap with another PR?
- [ x] The PR is self-contained and cannot be split into smaller PRs?
- [ x] Have I provided a clear explanation of the changes?
- [ ] Have I tested the changes (manually)?
    - [] Are all unit tests still passing? (`npm run test`)
- [x ] Are the changes visual?
  - [ x] Have I provided screenshots/videos of the changes?

### CODE REVIEW

**Comment by CodeTappert** [comment]
How is this localization?

**Comment by Tempo-anon** [comment]
This should also point to pagefaultgames:beta

File: src/enums/arena-tag-type.ts [hunk 1]
```diff
@@ -21,5 +21,7 @@ export enum ArenaTagType {
   MAT_BLOCK = "MAT_BLOCK",
   CRAFTY_SHIELD = "CRAFTY_SHIELD",
   TAILWIND = "TAILWIND",
+  // SAFEGUARD = "SAFEGUARD",
```
Comment (Tempo-anon) [inline src/enums/arena-tag-type.ts:24]: Why does this PR also comment out all safeguard stuff

Comment (DTZK) [reply src/enums/arena-tag-type.ts:24]: I made the safeguard code before this but then exams came so I commented it out from my repo before this

Comment (CodeTappert) [reply src/enums/arena-tag-type.ts:24]: Then remove the change from this PR

Comment (DTZK) [reply src/enums/arena-tag-type.ts]: > Then remove the change from this PR

I have done so

**Comment by Tempo-anon** [comment]
Not sure if this is needed for now since GMAX moves are not implemented. If we ever do decide to implement them we can do the change then.

**Pull request closed without merging**